style: added base styles for body, headings, links, text and images

almanbetzhorobekov opened the pull request:
css class

almanbetzhorobekov committed:
style: added base styles for body, headings, links, text and images

TimTAU commented on about-me.html:
```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
-<html lang="de-DE">
+<html lang="en">
```
Warum ist deine Seite jetzt auf Englisch? Der Inhalt ist doch weiterhin auf Deutsch

TimTAU commented on about-me.html:
```diff
@@ -1,15 +1,17 @@
     <meta charset="UTF-8">
     <title>Über mich</title>
     <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+    <link rel="stylesheet" href="styles.css">
```
Für die Images hast du es schon gemacht, ich würde auch die CSS Dateien in eine eigene Datei verschieben. Dann ist das für dich später ein wenig übersichtlicher

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	
+	background: #014464;
+	background: -moz-linear-gradient(top, #0272a7, #013953);
+	background: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#0272a7), to(#013953));
```
Hier hast du 3 mal die gleiche Property definiert. Schau mal im Inspektor des Browsers, ob das funktioniert
<img width="2549" height="468" alt="image" src="https://github.com/user-attachments/assets/47b625d4-ca47-4ac6-8294-604ac0136cc7" />

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	
+	-moz-border-radius: 10px;
+	-webkit-border-radius: 10px;
+	border-radius: 10px;
```
Can i use ist eine schöne Quelle, um zu prüfen, ob CSS Funktionen unterstützt sind. border-radius existiert in allen relevanten Browsern: https://caniuse.com/?search=border-radius
-moz-border-radius und -webkit-border-radius bspw. existieren gar nicht. Das sagt dir auch der Inspektor des Browsers (F12 > Inspektor)
<img width="399" height="115" alt="image" src="https://github.com/user-attachments/assets/99902937-b50d-45e3-9797-2be01946a418" />

TimTAU commented on menu.css:
Diese Datei wird nirgendwo referenziert

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+#menu {
+	list-style:none;
```
Was Auswirkung hat der List-Style an dieser Stelle?

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	border: 1px solid #002232;
+	-moz-box-shadow:inset 0px 0px 1px #edf9ff;
+	-webkit-box-shadow:inset 0px 0px 1px #edf9ff;
+	box-shadow:inset 0px 0px 1px #edf9ff;
```
Auch box-shadow existiert nicht in den verschiedenen Browser Varianten. Woher hast du die Information das hin zuschreiben? Hat bei dir etwas ohne die Angabe nicht funktioniert?

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	padding: 4px 10px 4px 10px;
+	margin-right:30px;
+	margin-top:7px;
+	border:none;
```
Welche Border wird hier zurück gesetzt?

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+}
+#menu li {
+	float:left;
+	display:block;
+	text-align:center;
+	position:relative;
```
Das Listen-Element hat die Größe deines Textes. Display block und text-align sollten daher keine Auswirkung haben. Du gibst die Position als relative an, hast aber keine Attribute die Positionieren. Brauchst du dann die Angabe?

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	border-radius: 5px 5px 0px 0px;
+}
+#menu li a {
+	font-family:Arial, Helvetica, sans-serif;
```
Sollen nur die Links im Menu diese Schriftart haben oder alle auf deiner Seite?

TimTAU commented on about-me.html:
```diff
@@ -21,19 +23,20 @@
+                Die Arbeit mit Kindern erfüllt mich jeden Tag aufs Neue. 
+                Ich bin stolz darauf, Teil dieses Kindergartens zu sein und genieße jede Minute mit den Kleinen.
+            </em></p>
         </p>
```
<img width="256" height="76" alt="image" src="https://github.com/user-attachments/assets/26b166ed-62db-4785-84b4-ddcbb1d93e7e" />

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	color:#161616;
+	text-shadow: 1px 1px 1px #FFFFFF;
+}
+#menu li .drop {
+	padding-right:21px;
+	background:url("img/drop.png") no-repeat right 8px;
+}
+#menu li:hover .drop {
+	background:url("img/drop.png") no-repeat right 7px;
```
Die beiden Bilder scheinen nicht in deinem Projekt enthalten zu sein

TimTAU commented on menu.css:
```diff
@@ -0,0 +1,70 @@
+	font-size:14px; 
+	color: #EEEEEE;
+	display:block;
+	outline:0;
+	text-decoration:none;
```
Dein CSS solltest du nach Bedarf hinzufügen. Ergänze eine Angabe und schau dir an welche Auswirkungen diese hat. Dann merkst du welche Angaben keine Auswirkungen haben und du kannst besser verstehen was durch die Angabe welcher Regel passiert

TimTAU commented on style.css:
Wofür ist style css und wofür ist styles css? Die Namen hören sich sehr ähnlich an

TimTAU commented on styles.css:
```diff
@@ -0,0 +1,121 @@
+  --bg-color: #f9f9f9;
+  --card-bg: #fdf6e3; 
+
+  --borfer-radius: 8px;
```
Typo
```suggestion
  --border-radius: 8px;
```

TimTAU commented on styles.css:
```diff
@@ -0,0 +1,121 @@
+
+  --borfer-radius: 8px;
+  --spacing: 20px;
+  --max-width: 1200px
```
Ohne das geht nix ;)
```suggestion
  --max-width: 1200px;
```

TimTAU commented on style.css:
Die beiden style Dateien habe ich mir jetzt erstmal nicht angeschaut. Mit den Kommentaren von oben kannst du diese denke ich schon selbst überarbeiten

almanbetzhorobekov committed:
1.implement basic layout and styling for about-me page
2.apply external stylesheet to about-me.html
3.add global CSS styling for about-me page

almanbetzhorobekov committed:
lang="DE" to lang="en"

almanbetzhorobekov committed:
deleted id="menu" and  class="bio"
